charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.6.7
 kubeVersion: ">=1.22.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.28.2
+version: 5.28.3
```

Comment:
Please bump minor chart version and update the changelog line on the bottom of this file

Reply:
Chart version updated to `5.28.3` https://github.com/argoproj/argo-helm/pull/1937/commits/96b627ee989d4ed68e6d72bfe4a012af50e14d29 as latest version of ArgoCD Helm Chart in `main` is `5.28.2`. Changelog updated  https://github.com/argoproj/argo-helm/pull/1937/commits/14621b4c713da7fb70235037716b284e783e4358.

Reply:
I wrote minor, not patch ;)
So as main is 5.28.2, you should bump to 5.29.0

Reply:
Fixed https://github.com/argoproj/argo-helm/pull/1937/commits/bf2e21b9e201491c6c463abe2af443839d944049.